versioned_docs/version-1.5.0/install-kubeslice/yaml/yaml-controller-install.mdx

```diff
@@ -0,0 +1,264 @@
+```
+
+From the above output, copy the URL for the **Kubernetes control plane** to
+add it as the cluster endpoint in the `controller.yaml` file.
+
+For example,
+`https://aks-controller-cluster-dns-06a5f5da.hcp.westus2.azmk8s.io:443`.
+
+### Controller Configuration Parameters
+The following tables describe the configuration parameters used to install the KubeSlice Controller.
+
+| Parameter                                                    | Parameter Type | Description                                                                  | Required  |
+| ------------------------------------------------------------ | -------------- | ---------------------------------------------------------------------------- | --------- |
+| Kubeslice                                                    | Object         | The cluster where the KubeSlice Controller is installed.                     | Mandatory |
+
+#### KubeSlice Parameters
+
+This parameter contains the configuration object used in the
+[KubeSlice Controller YAML file](#create-a-controller-yaml).
+
+| Parameter                            | Parameter Type | Description                                                            | Required  |
+| ------------------------------------ | -------------- | ---------------------------------------------------------------------- | --------- |
+| [controller](#controller-parameters) | Object         | The cluster where the KubeSlice Controller is installed.               | Mandatory |
+
+#### Controller Parameters
+This object contains the different parameters used in the [KubeSlice Controller YAML file](#create-a-controller-yaml).
+
+| Parameter          | Parameter Type | Description                                                                                        | Required  |
+| ------------------ | -------------- | -------------------------------------------------------------------------------------------------- | --------- |
+| loglevel           | String         | The log level of Controller. The default value is `INFO`. The other values are `DEBUG` or `ERROR`. | Optional  |
+| rbacResourcePrefix | String         | The RBAC resource prefix.                                                                          | Optional  |
+| projectnsPrefix    | String         | The project namespace prefix.                                                                      | Optional  |
+| endpoint           | AlphaNumeric   | The URL of the Kubernetes control plane.                                                           | Mandatory |
+
+
+### Create a Controller YAML
+Create the `controller.yaml` file using the following template.
+
+:::info
+To understand more about the parameters, see [Controller Configuration Parameters](#controller-configuration-parameters).
+:::
+
+```
+kubeslice:
+kubeslice:
+   # user can configure labels or annotations that Kubetally Controller namespaces should have
+   namespaceConfig:
```

Comment:
Just asking - The namespaceConfig object and its parameters are not described under Controller Configuration Parameters above.

Reply:
done